int32 event_id
int32 result_id
float64 start_time
float64 duration_time
string comment
